ui/raidboss/data/04-sb/raid/o1n.txt

```diff
@@ -0,0 +1,148 @@
+424.0 "Wyrm Tail" sync /:Alte Roite:23D6:/
+431.1 "Charybdis" sync /:Alte Roite:23DB:/
+438.2 "Roar" sync /:Alte Roite:23DC:/
+444.3 "Wyrm Tail" sync /:Alte Roite:23D6:/
+452.0 "Twin Bolt x2" # sync /:Alte Roite:23D8:/
+456.5 "Clamp" sync /:Alte Roite:23E2:/
+462.6 "Roar" sync /:Alte Roite:23DC:/
+
+473.0 "The Classical Elements" sync /:Alte Roite:23E0:/
+476.2 "Flame" sync /:Alte Roite:23DD:/
+479.3 "Flame" sync /:Alte Roite:23DD:/
+484.4 "Flash Freeze" sync /:Alte Roite:23DF:/
+494.5 "Downburst" sync /:Alte Roite:1ED8:/
+495.3 "Burn" sync /:Ball Of Fire:23D5:/
+498.8 "Burn" sync /:Ball Of Fire:23D5:/
+504.7 "Blaze" sync /:Alte Roite:23E1:/
+516.8 "Charybdis" sync /:Alte Roite:23DB:/ window 30,30
+523.9 "Roar" sync /:Alte Roite:23DC:/
+526.0 "Flame" sync /:Alte Roite:23DD:/
+528.1 "Flash Freeze" sync /:Alte Roite:23DF:/
+538.2 "Downburst" sync /:Alte Roite:1ED8:/
+545.5 "Burn" sync /:Ball Of Fire:23D5:/
+549.6 "Levinbolt" sync /:Alte Roite:23DA:/ window 30,30
+556.2 "Twin Bolt x2" # sync /:Alte Roite:23D8:/
+561.7 "Wyrm Tail" sync /:Alte Roite:23D6:/
+567.8 "Wyrm Tail" sync /:Alte Roite:23D6:/
+575.5 "Twin Bolt x2" # sync /:Alte Roite:23D8:/
+581.0 "Flame" sync /:Alte Roite:23DD:/
+584.1 "Flame" sync /:Alte Roite:23DD:/
+599.5 "Breath Wing" sync /:Alte Roite:23DE:/ window 30,30
+600.2 "Burn" sync /:Ball Of Fire:23D5:/
+603.6 "Burn" sync /:Ball Of Fire:23D5:/
+606.6 "Roar" sync /:Alte Roite:23DC:/
+612.8 "Wyrm Tail" sync /:Alte Roite:23D6:/
+618.9 "Wyrm Tail" sync /:Alte Roite:23D6:/
+626.0 "Charybdis" sync /:Alte Roite:23DB:/ window 30,30
+633.1 "Roar" sync /:Alte Roite:23DC:/
+639.2 "Wyrm Tail" sync /:Alte Roite:23D6:/
+646.9 "Twin Bolt x2" # sync /:Alte Roite:23D8:/
+651.4 "Clamp" sync /:Alte Roite:23E2:/
+657.5 "Roar" sync /:Alte Roite:23DC:/
+
+667.9 "The Classical Elements" sync /:Alte Roite:23E0:/
+671.1 "Flame" sync /:Alte Roite:23DD:/
+674.2 "Flame" sync /:Alte Roite:23DD:/
+679.3 "Flash Freeze" sync /:Alte Roite:23DF:/
+689.4 "Downburst" sync /:Alte Roite:1ED8:/
+690.2 "Burn" sync /:Ball Of Fire:23D5:/
+693.6 "Burn" sync /:Ball Of Fire:23D5:/
+699.5 "Blaze" sync /:Alte Roite:23E1:/
+711.7 "Charybdis" sync /:Alte Roite:23DB:/ window 30,30
+718.8 "Roar" sync /:Alte Roite:23DC:/
+720.9 "Flame" sync /:Alte Roite:23DD:/
+723.0 "Flash Freeze" sync /:Alte Roite:23DF:/
+733.1 "Downburst" sync /:Alte Roite:1ED8:/
+740.4 "Burn" sync /:Ball Of Fire:23D5:/
+744.5 "Levinbolt" sync /:Alte Roite:23DA:/ window 30,30
+751.1 "Twin Bolt x2" # sync /:Alte Roite:23D8:/
+756.6 "Wyrm Tail" sync /:Alte Roite:23D6:/
+762.7 "Wyrm Tail" sync /:Alte Roite:23D6:/
+769.8 "Charybdis" sync /:Alte Roite:23DB:/
+776.9 "Roar" sync /:Alte Roite:23DC:/ window 30,15
+783.0 "Wyrm Tail" sync /:Alte Roite:23D6:/
+790.7 "Twin Bolt x2" # sync /:Alte Roite:23D8:/
+795.2 "Clamp" sync /:Alte Roite:23E2:/
+801.3 "Roar" sync /:Alte Roite:23DC:/
+803.5 "Flame" sync /:Alte Roite:23DD:/ window 50,5
+820.7 "Burn Enrage" sync /:Ball Of Fire:23E4:/
```

Comment:
Yea, this seems reasonable to roll the whole loop out if it's a fixed time and there's a fixed enrage.